sherlock/resources/data.json

```diff
@@ -2447,12 +2447,13 @@
     "username_unclaimed": "noonewouldeverusethis77777"
   },
   "koo": {
-    "errorMsg": "Invalid Profile",
+    "errorMsg": "handle {} is available",
```

Comment:
Two minor issues here:
1. "is available" is not an `errorMsg`. "already taken" would be an error message as that is the message for when a username is not found
2. I see that you have used `{}` in the `errorMsg`. Sherlock won't format the string with the username. So it is better to just use `already taken` as the `errorMsg`.

In other words, replace the value for `errorMsg` with `already taken`

Reply:
Sorry my bad. It should just be `is available`